### PR TITLE
Map 625 uof new weapons observed question

### PR DESCRIPTION
**Pull request by GurnankCheema**
Add new 'weapons observed' question to the report details page

The functionality for this is identical to the body-worn camera question. So the macro previously used for body-worn cameras has been adapted to make it reusable for both questions.

### CODE REVIEW

File: server/views/pages/reportDetailMacro.njk [hunk 1]
```diff
@@ -214,6 +214,18 @@
         print: print
       })
     }}
+
```
Comment (GurnankCheema) [inline server/views/pages/reportDetailMacro.njk:217]: the 'if' is needed for old reports that predate the weapons question

File: server/config/forms/useOfForceDetailsForm.js [hunk 2]
```diff
@@ -53,6 +52,27 @@ const completeSchema = joi.object({
 
   pavaDrawn: requiredBooleanMsg('Select yes if PAVA was drawn').alter(optionalForPartialValidation),
```
Comment (GurnankCheema) [inline server/config/forms/useOfForceDetailsForm.js:54]: validation spec for the new weapons observed question

File: server/views/formPages/incidentMacros.njk [hunk 3]
```diff
@@ -51,94 +51,6 @@
 </div>
 {% endmacro %}
 
-
-{% macro radioWithMultipleNestedTextBox(question) %}
```
Comment (GurnankCheema) [inline server/views/formPages/incidentMacros.njk:55]: this macro renamed and rewritten

File: server/views/formPages/incidentMacros.njk [hunk 4]
```diff
@@ -394,7 +380,7 @@
                <div class="govuk-checkboxes__item">
                    <input
                            class="govuk-checkboxes__input"
-                           id="parent-{{option.value}}"
```
Comment (GurnankCheema) [inline server/views/formPages/incidentMacros.njk:397]: fix a label attribute clash

**Review by danbenton-mojdt — CHANGES_REQUESTED** [review]
No uniqueness required on list of weapons observed

**Review by danbenton-mojdt — APPROVED** [review]

**Pull request merged**